poc_iot_injector/src/server.rs

```diff
@@ -17,29 +20,32 @@ pub const DEFAULT_LAST_POC_SUBMISSION_TS_SECS: i64 = 0000000000;
 
 pub struct Server {
     pool: Pool<Postgres>,
-    keypair: Keypair,
+    keypair: Arc<Keypair>,
     txn_service: TransactionService,
     iot_verifier_store: FileStore,
     last_poc_submission_ts: MetaValue<i64>,
 }
 
 impl Server {
     pub async fn new(pool: Pool<Postgres>, keypair: Keypair) -> Result<Self> {
-        // TODO: Find this time in meta store db first
-        let last_poc_submission_ts = env::var("LAST_POC_SUBMISSION_TS")
+        let env_last_poc_submission_ts = env::var("LAST_POC_SUBMISSION_TS")
```

Comment:
@JayKickliter had the good suggestion to me earlier that ambiguous envvars (poc being a pretty broad category for us) might want app or network name spacing; avoid future conflicts with any other services running that might want a similar environment variable?

Reply:
Please see https://github.com/helium/oracles/pull/142 for where this is going